server/tso/allocator_manager.go

```diff
@@ -75,6 +77,9 @@ type AllocatorManager struct {
 		allocatorGroups map[string]*allocatorGroup
 		// dc-location/global (string) -> Member ID
 		clusterDCLocations map[string][]uint64
+		// suffixDCLocations is a map of the dc-locations to its TSO suffix.
+		// dc-location (string) -> Suffix number (int)
+		suffixDCLocations map[string]int32
```

Comment:
Can we merge `clusterDCLocations` and `suffixDCLocations` as one to prevent inconsistency here?